.env.dist

```diff
@@ -37,5 +37,13 @@ DATABASE_URL=mysql://${DOCKER_MYSQL_USER}:${DOCKER_MYSQL_PASSWORD}@${DOCKER_MYSQ
 # For Gmail as a transport, use: "gmail://username:password@localhost"
 # For a generic SMTP server, use: "smtp://localhost:25?encryption=&auth_mode="
 # Delivery is disabled by default via "null://localhost"
-MAILER_URL=null://localhost
+MAILER_URL=
+MAILER_USERNAME=
+MAILER_PASSWORD=
+MAILER_TRANSPORT=
+MAILER_HOST=
+MAILER_ENCRYPTION=
+MAILER_PORT=
+MAILER_FROM=
+
```

Comment:
This would be for the S2, but I think AWS has a credentials system where you store them through their website, and it injects them in your `.env` (at runtime) so no-one sees them, and you don't have to handle your own local `.env`. I'll do some research when we have some time.

Reply:
Okay, we will talk later about this